xbmc/video/dialogs/GUIDialogVideoInfo.h

```diff
@@ -107,11 +107,11 @@ class CGUIDialogVideoInfo :
   static bool OnGetFanart(const CFileItemPtr &videoItem);
 
   CFileItemPtr m_movieItem;
-  CFileItemList *m_castList;
-  bool m_bViewReview;
-  bool m_bRefresh;
-  bool m_bRefreshAll;
-  bool m_hasUpdatedThumb;
-  bool m_hasUpdatedUserrating;
-  int m_startUserrating;
+  CFileItemList *m_castList = new CFileItemList;
+  bool m_bViewReview = false;
+  bool m_bRefresh = false;
+  bool m_bRefreshAll = true;
+  bool m_hasUpdatedThumb = false;
+  bool m_hasUpdatedUserrating = false;
+  int m_startUserrating = -1;
```

Comment:
So should I remove https://github.com/xbmc/xbmc/pull/10889/commits/9da6022ff80c144992ea95433e90059d6e795091 again?